dalton-agent/dalton-agent.py

```diff
@@ -1114,7 +1114,7 @@ def check_for_errors(tech):
         ids_log_fh = open(JOB_IDS_LOG, "r")
         for line in ids_log_fh:
             if tech.startswith('suri'):
-                if ("<Error>" in line or line.startswith("ERROR") or line.startswith("Failed to parse configuration file")):
+                if ("<Error>" in line or "Error: " in line or line.startswith("ERROR") or line.startswith("Error: ") or line.startswith("Failed to parse configuration file")):
```

Comment:
Thanks for the PR.

One nit here ... if you are checking for `"Error: " in line` then checking for `line.startswith("Error: ")` is redundant and superfluous.

Reply:
Ah, yup.  That the result of catching in one for socket mode (which includes some stuff infront) and one for non-socket mode and then not removing it once I nailed it down.  I will get it fixed up and PR updated